README.md

```diff
@@ -1,2 +1,54 @@
 # client-cpp
 A client library for C++
+
+#Pre-requisites
+
+###Compiler
+The library requires C++14 standard, thus GCC 5, or Clang 3.4 are required.
+
+###Libraries
+* [CMake](https://cmake.org/)
+* [Boost](http://www.boost.org/)
+* [POCO](https://pocoproject.org/)
+
+####Third party libraries and tools installation
+##### Linux
+    $ [sudo] apt-get install autoconf automake libtool build-essential  
+    $ [sudo] apt-get install libboost-all-dev libpoco-dev libgflags-dev libgtest-dev curl unzip doxygen g++-5
+
+##### Mac OS X
+    $ [sudo] xcode-select --install
+    $ [sudo] brew autoconf automake libtool build-essential
+    $ [sudo] brew install libboost-all-dev libpoco-dev libgflags-dev libgtest-dev curl unzip doxygen g++-5
+
+#### gRPC and Protobuf
+Google's [gRPC](https://www.grpc.io) and [Protobuf](https://developers.google.com/protocol-buffers/) 
+are the core components of the library, and the latest release versions should be present in the system
```

Comment:
`latest release versions should be present in the system` requires additional explanation. 

I would append this sentence with something like `Each platform defines its particular rules ... <bla-bla-bla>.` and then add more details (on this "presence") to the each of the OS-related sections below.